Lib/test/test_complex.py

```diff
@@ -597,6 +598,8 @@ def test_abs(self):
         for num in nums:
             self.assertAlmostEqual((num.real**2 + num.imag**2)  ** 0.5, abs(num))
 
+        self.assertRaises(OverflowError, abs, complex(*[DBL_MAX]*2))
```

Comment:
Nitpick: that seems unnecessarily convoluted.
```suggestion
        self.assertRaises(OverflowError, abs, complex(DBL_MAX, DBL_MAX))
```